__tests__/components/HeaderSystemMenu.test.js

```diff
@@ -23,51 +23,144 @@ SOFTWARE.
 import { shallow } from 'enzyme';
 import HeaderSystemMenu from '../../src/header/HeaderSystemMenu';
 import setLanguage from '../../src/functions/setLanguage';
+import { useMenuHandler } from '@bluecateng/pelagos';
 
 jest.unmock('../../src/header/HeaderSystemMenu');
 
 jest.mock('../../src/hooks/usePlatformData', () =>
     jest
         .fn()
         .mockReturnValueOnce({
-            data: { user: { permissions: { download_logs: false } } },
+            data: {
+                user: {
+                    permissions: { download_logs: false, view_logs: false },
+                },
+            },
         })
         .mockReturnValueOnce({
-            data: { user: { permissions: { download_logs: true } } },
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
         })
         .mockReturnValueOnce({
-            data: { user: { permissions: { download_logs: true } } },
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
+        })
+        .mockReturnValueOnce({
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
+        })
+        .mockReturnValueOnce({
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
+        })
+        .mockReturnValueOnce({
```

Comment:
I see you added two new test cases but you are returning the new mock values 3 times here now. Do we need the last `mockReturnValueOnce` ?

Reply:
Youre right, one extra.